book/07-git-tools/sections/stashing-cleaning.asc

```diff
@@ -277,3 +277,10 @@ What now>
 ----
 
 This way you can step through each file individually or specify patterns for deletion interactively.
+
+[NOTE]
+====
+There is a quirky situation where you might need to be extra forceful in asking Git to clean your working directory. 
+It's possible that you have a working directory wherein one or more subdirectories are _themselves_ Git clones, which means you have subdirectories that contain their own `.git` directories.
+Even with the "force" `-f` option, Git will not remove ignored subdirectories that contain either a file or directory named `.git` -- to get rid of those, you need to use _two_ `-f` options for emphasis.
+====
```

Comment:
🤔 The way you're describing this isn't really a supported use case in Git. What do you think about this?
```
If you happen to clone or copy a Git repository into the working directory of another one, even `git clean -fd` will refuse to delete it.
In this situation, you have to provide a second `-f` flag to get rid of it.
```

Reply:
OK, I'll deal with that this evening.

Reply:
OK, I compromised, the new wording is somewhere in between.